Formula/b/bandicoot.rb

```diff
@@ -0,0 +1,42 @@
+class Bandicoot < Formula
+  desc "C++ library for GPU accelerated linear algebra"
+  homepage "https://coot.sourceforge.io/"
+  url "https://gitlab.com/conradsnicta/bandicoot-code/-/archive/1.11.0/bandicoot-code-1.11.0.tar.bz2"
+  sha256 "e53fa0db2cd890475ad41452b0a70b036315e32dcbe45687f149789e341a85f6"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+    regex(%r{url=.*?/bandicoot-code[._-]v?(\d+(?:\.\d+)+)\.t}i)
+  end
```

Comment:
```suggestion

```

Reply:
The [default `Git` strategy](https://rubydoc.brew.sh/Homebrew/Livecheck/Strategy/Git.html) works just fine here, so we can omit this.

Reply:
Okay, thanks for the heads up! I'll remove it.